enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -72,6 +72,7 @@ <h2 class="course-confirmation-title">{{ confirmation_text }}</h2>
                 {{ price_text }}:
                 {% if course_mode.final_price and course_mode.original_price != course_mode.final_price %}
                   <strike>{{ course_mode.original_price }}</strike> {{ course_mode.final_price }}
+                  <div>{{discount_text}} <b>{{enterprise_customer.name}}</b></div>
```

Comment:
See below: we can probably make this just `{discount_text}` if we pass in the full string at once.